### ISSUE TITLE
WCAG 1.4.11 contrast kleuren

### ISSUE THREAD

**Issue by commonpike** [ProDemos/pds#147]
- https://github.com/ProDemos/stem.scholierenverkiezingen.nl/issues/377#issue-3671310493

> De grijze kleur van de invoervelden zoals bijvoorbeeld "E-mail*" hebben met de witte achtergrond een contrastverhouding van 1.1:1 in plaats van minimaal 3.0:1.

<img width="480" height="368" alt="Image" src="https://github.com/user-attachments/assets/3659c967-126b-41dd-bda3-610260c130ed" />

<img width="805" height="373" alt="Image" src="https://github.com/user-attachments/assets/28dfc129-1b09-491f-b7f7-5aa991ff4016" />

maar dat geldt dan ook voor de knoppen, en voor alle ProDemos sites ..

**Comment by commonpike** [comment]
@sdevogel , wat vind jij ervan ?

**Comment by sdevogel** [comment]
Ja lastig...

Ik denk dat we er een border omheen moeten plaatsen die wel genoeg contrast heeft. Ik heb hier volgens mij ooit al eens naar gekeken omdat in mijn sketch document de invoervelden een rand hebben creme-110 (#DFD7CC). Maar helaas is dit ook niet genoeg, en creme-120 is geeft ook nog onvoldoende contrast.

<img width="1427" height="855" alt="Image" src="https://github.com/user-attachments/assets/2a620497-f89c-4f62-bc4d-070d564a3451" />

Het invoerveld hieronder is een border met grijswaarde (#949494) welke nét genoeg contrast geeft (3.03:1) maar is niet echt onderdeel van de huisstijl en best wel hard maar misschien wel de oplossing?

<img width="523" height="135" alt="Image" src="https://github.com/user-attachments/assets/a2fa38f9-c01a-42f4-8cd1-356a51899e54" />

**Comment by commonpike** [comment]
#949494 werkt alleen op een witte achtergrond. 
PDS input velden zijn creme; #949494 vs #f5f0e9 is 2.64 : 1, niet genoeg. 
black-60 zou het wel doen.

als we de achtergrondkleur gaan veranderen moeten we ook gaan kijken naar de placeholder text enzo ... 

<img width="338" height="111" alt="Image" src="https://github.com/user-attachments/assets/39e4e16f-0874-4b64-86f2-953c59cd171b" />


https://pds.prodemos.nl/components/forms.html

**Comment by commonpike** [comment]
Om door de keuring te komen ga ik dit doen

```
// add a dark beige outline on non-focused elements for
// accessible contrast. but on focus, hide the 
// outline since there is a 3px border there in pds
:root {
    --pds-color-formstroke: #8F8275;
}
.pds-c-input[type=text], .form-group [type=text].form-control, 
.pds-c-input[type=number], .form-group [type=number].form-control, 
.pds-c-input[type=email], .form-group [type=email].form-control, 
.pds-c-input[type=tel], .form-group [type=tel].form-control, 
.pds-c-input[type=password], .form-group [type=password].form-control, 
.pds-c-input[type=number], .pds-c-input[type=date], .form-group [type=date].form-control, 
.pds-c-input[type=time], .form-group [type=time].form-control, 
.pds-c-input.text, .form-group .text.form-control {
    transition: box-shadow 300ms ease-in-out;
    &::placeholder {
        color:var(--pds-color-formstroke);
    }
    &:not(:focus), &:disabled {
        box-shadow: 0 0 0 1px var(--pds-color-formstroke);
        
    }
}

select.pds-c-input, .form-group select.form-control,
.pds-c-input.dropdown, .form-group .dropdown.form-control {
    transition: box-shadow 300ms ease-in-out;
    &::placeholder {
        color:var(--pds-color-formstroke);
    }
    &:not(:focus):not(:hover), &:disabled {
        box-shadow: 0 0 0 1px var(--pds-color-formstroke);
    }
}



```


<img width="819" height="617" alt="Image" src="https://github.com/user-attachments/assets/dda1f564-ccc3-4d21-ab48-ce546b53794f" />

<img width="951" height="490" alt="Image" src="https://github.com/user-attachments/assets/24b0c0df-f357-4373-a780-ee740fd0c751" />

**Comment by commonpike** [comment]
maar echt consequent is het niet :-(

<img width="938" height="530" alt="Image" src="https://github.com/user-attachments/assets/72d01cdc-9c20-42a0-ac93-bc376b13ce3c" />

**Comment by sdevogel** [comment]
Hey Pike, ik zou dit voor nu doen.
Mijn voornemen is om in 2027 PDS onder de loep te nemen waar we dit dan in mee nemen.
